content/oci/oci.go

```diff
@@ -467,6 +538,17 @@ func (s *unsafeStore) Predecessors(ctx context.Context, node ocispec.Descriptor)
 	return s.graph.Predecessors(ctx, node)
 }
 
+// isContextDone returns an error if the context is done.
+// Reference: https://pkg.go.dev/context#Context
+func isContextDone(ctx context.Context) error {
+	select {
+	case <-ctx.Done():
+		return ctx.Err()
```

Comment:
This path can be tested by having a context cancelled immediately.

Basically, you need unit test cases for `isContextDone()`.

Reply:
Added unit tests.